sim/hunter/serpent_sting.go

```diff
@@ -59,7 +59,7 @@ func (hunter *Hunter) registerSerpentStingSpell() {
 			Tag:      "SerpentSting",
 			ActionID: actionID,
 			OnGain: func(aura *core.Aura, sim *core.Simulation) {
-				hunter.AttackTables[aura.Unit.UnitIndex].DamageDealtMultiplier *= noxiousStingsMultiplier
+				hunter.AttackTables[aura.Unit.UnitIndex].DamageTakenMultiplier *= noxiousStingsMultiplier
```

Comment:
Repeating from an earlier comment, but I don't think this change should be necessary. The fact that we're retrieving the specific attack table already implies a attacker --> defender relationship, so there only needs to be 1 damage multiplier value.

Reply:
See above: taken and dealt multipliers work differently, mostly wrt DoTs